src/Microsoft.OData.Client/EntityTracker.cs

```diff
@@ -26,12 +26,12 @@ public class EntityTracker : EntityTrackerBase
 
         #region Resource state management
 
-    #if PORTABLELIB && WINDOWSPHONE
+#if PORTABLELIB && WINDOWSPHONE

```

Comment:
@ElizabethOkerio For some time now, we have been getting rid of these `PORTABLELIB` blocks. I think you should get rid of this too